quence.lua

```diff
@@ -615,13 +615,15 @@ function grid_device.key(x, y, z)
             press = coord
         end
 
+
         -- scalar transpose down (row 6, col 12)
         if coord == 1206 then
             for step = 1, seqlen[page] do
                 steps[page][step] = steps[page][step] - 1
-                if steps[page][step] < 1 then
-                    steps[page][step] = #scale
-                end
+                --if steps[page][step] < 1 then
```

Comment:
My advice is to just delete old code; keeping track of it in case you might want to restore it at some point is exactly what git is for :)

Reply:
good advice. thanks!